AppHandling/Get-TestsFromNavContainer.ps1

```diff
@@ -18,6 +18,10 @@
   If your container is running AAD authentication, you need to specify an accesstoken for the user specified in credential
  .Parameter testSuite
   Name of test suite to get. Default is DEFAULT.
+ .Parameter ExtensionId
+  Specifying an extensionId causes the test tool to run all tests in the app with this app id.

```

Comment:
Doc here seems inaccurate. The function is called `Get-TestsFromBcContainer` so these parameters are used for filtering, there is no test running here.